mlflow/entities/run_info.py

```diff
@@ -24,7 +24,7 @@ class RunInfo(_MLflowObject):
     DELETED_LIFECYCLE = "deleted"
 
     def __init__(self, run_uuid, experiment_id, name, source_type, source_name, entry_point_name,
-                 user_id, status, start_time, end_time, source_version, lifecycle_stage,
+                 user_id, status, start_time, end_time, source_version, lifecycle_stage=None,
```

Comment:
Realized we should make `lifecycle_stage` optional / have a default value for it to allow for backwards-compatibility with already-persisted runs

Reply:
Isn't this already handled by `_read_persisted_run_info_dict` in `file_store.py`?

Reply:
Ah good catch yeah it is, I can remove the change from this PR